ansible/tasks/deploy_stack.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Copying docker-compose.yml file on the server
+  copy:
+    src: "/opt/tools/app/docker-compose.yml"
+    dest: "/home/{{ ansible_user }}/docker-compose.yml"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    mode: u=rw,g=rw,o=r
+
+
+- name: Create a directory if it does not exist
+  file:
+    path: /opt/sc4s
+    state: directory
+    mode: '0755'
+
+- name: Create a directory if it does not exist
+  file:
+    path: /opt/sc4s/tls
+    state: directory
+    mode: '0755'
+
+- name: Create a directory if it does not exist
```

Comment:
Create SC4S/archive directory if it doesn't exist

Reply:
fixed.